frontend/apps/reader/readerui.lua

```diff
@@ -497,6 +516,10 @@ function ReaderUI:showReader(file, provider)
         self:showFileManager(file)
         return
     end
+
+    -- We can now signal the existing ReaderUI/FileManager instances that it's time to go bye-bye...
+    UIManager:broadcastEvent(Event:new("ShowReader"))
```

Comment:
I would read this event name "ShowReader" as a request to show the reader - and I don't think that's what it does.
It feels like it should be named "ReaderShown" or "ReaderShowing".